tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -870,10 +878,10 @@ func (az *AzureProvider) constructNetworkProfile(ctx context.Context,
 		ipConfigName := az.generateResourceName("ipConfig", vmRandString)
 		az.NicName = nicName
 		ptrvmNic, err := az.createNIC(ctx, vnetName, subnetName, nsgName, ipConfigName)
```

Comment:
ptrvmNic -> can this be renamed to something that is more understandable and follows go naming conventions?

Reply:
Given the amount of technical debt we have in this repo, I am fine changing things when are making changes in the area. This of course is withing reason :-)